### PR TITLE
add enum baudrates

### PR DESCRIPTION
**Pull request by OwenBrotherwood**
fixes perceived limitation of available baud rates https://github.com/Microsoft/pxt/issues/1752

### CODE REVIEW

File: libs/core/serial.cpp [hunk 1]
```diff
@@ -15,8 +15,26 @@ enum SerialPin {
...
   //% block=9600
   BaudRate9600 = 9600
+  //% block=4800
+  BaudRate4800 = 4800,    
+  //% block=2400
+  BaudRate2400 = 2400,    
+  //% block=1200
+  BaudRate1200 = 1200,    
+  //% block=300
+  BaudRate300 = 300,
```
Comment (pelikhan) [inline libs/core/serial.cpp]: spurious comma I think

**Comment by pelikhan** [comment]
enum.d.ts needs to be regenerated

**Comment by OwenBrotherwood** [comment]
enum.d.ts and core-strings.json coming soon
I double check to see what else I missed

**Comment by OwenBrotherwood** [comment]
Order of baudrates isn't numeric: hope this is ok
image

**Comment by pelikhan** [comment]
That's ok.

**Comment by OwenBrotherwood** [comment]
Side note while I remember, enum.d.ts and core-strings.json after changes of serial.cpp where created with pxt serve on a git clone pxt-microbit with npm install -g pxt.
I investigate the pxt serve -yt / pxt buildtarget after I do some fix up on this disastrous PR

**Comment by OwenBrotherwood** [comment]
:( My Windows Yotta is bust: I cannot check with pxt buildtarget ( pxt serve -yt).
However, it seems that the files where generated without yotta (enum.d.ts and core-strings.json)
And, if the tests are good, all checks have passed.

**Pull request merged**